style.css

```diff
@@ -129,6 +129,12 @@ body {
   border-radius: 10px;
   font-size: 13px;
   margin-right: 10px;
+  transition: all 4s;
+}
+
+.skills:hover {
+  background-color: #6070ff;
```

Comment:
- [ ] Kindly change the title to be something more descriptive instead of being general like "Arnaud's Portfolio".
https://github.com/B77748/arnaud_portfolio/blob/37140a8ed7681fc57703d8af950cd692f1b05792/index.html#L12